### PR TITLE
blastp: add page

### PR DESCRIPTION
**Pull request by koolape**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.12.0

### CODE REVIEW

File: pages/linux/blastp.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# blastp
+
+> Protein-Protein BLAST.
+> More information: <https://www.ncbi.nlm.nih.gov/books/NBK279684/table/appendices.T.blastp_application_options/>.
+
+- Display help (use `-help` for detailed help):
```
Comment (kbdharun) [inline pages/linux/blastp.md:6]: Can you move this command to last?

Comment (koolape) [reply pages/linux/blastp.md]: ok

File: pages/linux/blastp.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Align two or more sequences using blastp-fast:
+
+`blastp -task blastp-fast -query {{query.fa}} -subject {{subject.fa}}`
+
+- Align two or more sequences, custom tabular output format, output to file:
+
+`blastp -query {{query.fa}} -subject {{subject.fa}} -outfmt {{'6 qseqid qlen qstart qend sseqid slen sstart send bitscore evalue pident'}} -out {{output.tsv}}`
+
+- Search protein databases using a protein query, 16 threads (CPUs) to use in the BLAST search, with a maximum number of 10 aligned sequences to keep:
```
Comment (navarroaxel) [inline pages/linux/blastp.md:18]:
```suggestion
- Search protein databases using a protein query, 16 threads to use in the BLAST search, with a maximum number of 10 aligned sequences to keep:
```

File: pages/linux/blastp.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`blastp -query {{query.fa}} -subject {{subject.fa}} -evalue {{1e-9}}`
+
+- Align two or more sequences using blastp-fast:
+
+`blastp -task blastp-fast -query {{query.fa}} -subject {{subject.fa}}`
+
+- Align two or more sequences, custom tabular output format, output to file:
+
+`blastp -query {{query.fa}} -subject {{subject.fa}} -outfmt {{'6 qseqid qlen qstart qend sseqid slen sstart send bitscore evalue pident'}} -out {{output.tsv}}`
```
Comment (navarroaxel) [inline pages/linux/blastp.md:16]: We use the quotes outside the token syntax
```suggestion
`blastp -query {{query.fa}} -subject {{subject.fa}} -outfmt '{{6 qseqid qlen qstart qend sseqid slen sstart send bitscore evalue pident}}' -out {{output.tsv}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**